Ginger/Ginger/ALM/Qtest/QtestCyclesExplorerPage.xaml.cs

```diff
@@ -0,0 +1,239 @@
+    {
+        public enum eExplorerTestLabPageUsageType { Import, Select, BrowseFolders }
+
+        ObservableList<BusinessFlow> mBizFlows;
+        private List<string[]> mExecDetailNames;
+        ObservableList<QTestApiModel.TestCycleResource> treeData;
+
+        private ITreeViewItem mCurrentSelectedTreeItem = null;
+        public string CurrentSelectedPath { get; set; }
+        ObservableList<QtestSuiteTreeItem> mCurrentSelectedTestSuites = new ObservableList<QtestSuiteTreeItem>();
+        object mCurrentSelectedObject = new object();
+
+        public ObservableList<QtestSuiteTreeItem> CurrentSelectedTestSuites
+        {
+            get
+            {
+                return mCurrentSelectedTestSuites;
+            }
+            set
+            {
+                mCurrentSelectedTestSuites = value;
+            }
+        }
+        private string mImportDestinationPath = string.Empty;
+        private bool mParentSelectionMode = false;
+        GenericWindow _GenericWin = null;
+
+        public QtestCyclesExplorerPage(string importDestinationPath = "", bool parentSelectionMode = false)
+        {
+            InitializeComponent();
+
+            mImportDestinationPath = importDestinationPath;
+            mParentSelectionMode = parentSelectionMode;
+
+            GetTreeData();
+
+            QtestCyclesExplorerTreeView.TreeTitle = "'" + ALMCore.AlmConfig.ALMDomain + " \\ " + ALMCore.AlmConfig.ALMProjectName + "' - Test Cycles Explorer";
+            QtestCyclesExplorerTreeView.TreeTitleStyle = (Style)TryFindResource("@ucTitleStyle_3");
+            QtestCyclesExplorerTreeView.Tree.ItemSelected += TestLabExplorerTreeView_ItemSelected;
+            foreach (QTestApiModel.TestCycleResource cycle in treeData)
+            {
+                QtestCycleTreeItem tvi = new QtestCycleTreeItem(cycle.TestCycles, cycle.TestSuites);
+                tvi.Name = cycle.Name.ToString();
+                tvi.ID = cycle.Id.ToString();
+                tvi.Path = cycle.Path.ToString();
+                QtestCyclesExplorerTreeView.Tree.AddItem(tvi);
+            }
+
+            LoadDataBizFlows();
+
+            mExecDetailNames = new List<string[]>();
+
+            ShowTestSetDetailsPanel(false);
+        }
+
+        private void GetTreeData()
+        {
+            Mouse.OverrideCursor = Cursors.Wait;
+            treeData = QtestConnect.Instance.GetQTestCyclesByProject(WorkSpace.Instance.Solution.ALMServerURL, WorkSpace.Instance.UserProfile.ALMUserName, WorkSpace.Instance.UserProfile.ALMPassword, WorkSpace.Instance.Solution.ALMProject);
+            Mouse.OverrideCursor = null;
+        }
+
+        private void LoadExecutionDetails()
+        {
+            TSExecDetails.Children.Clear();
+            //Total TC's
+            int total = 0;
+            foreach (string[] detail in mExecDetailNames)
+            {
+                short number;
+                if (Int16.TryParse(detail[1], out number))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)